.github/workflows/dev-release-grpc-service-pypi-publish.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     needs: [collect-version]
     uses: th2-net/.github/.github/workflows/compaund-python-grpc-pypi-publication.yml@main
     with:
-      custom-version: ${{ needs.collect-version.outputs.versionNumber }}-dev
+      custom-version: ${{ needs.collect-version.outputs.versionNumber }}-rc1
```

Comment:
@Nikita-Smirnov-Exactpro The version should look like `1.1.2rc1` according to [version format supported by python](https://peps.python.org/pep-0440/#public-version-identifiers)